op-batcher/batcher/driver.go

```diff
@@ -413,15 +416,15 @@ func (l *BatchSubmitter) sendTransaction(ctx context.Context, txdata txData, que
 	return nil
 }
 
-func (l *BatchSubmitter) blobTxCandidate(data []byte) (*txmgr.TxCandidate, error) {
-	l.Log.Info("building Blob transaction candidate", "size", len(data))
-	var b eth.Blob
-	if err := b.FromData(data); err != nil {
-		return nil, fmt.Errorf("data could not be converted to blob: %w", err)
-	}
+func (l *BatchSubmitter) blobTxCandidate(data txData) (*txmgr.TxCandidate, error) {
+	blobs := data.Blobs()
+	size := data.Len()
+	l.Log.Info("building Blob transaction candidate", "size", size, "num_blobs", len(blobs))
+	l.Metr.RecordBlobUsedBytes(size)
```

Comment:
This metric was intended to allow us to observe how close we are to filling up single blobs.   Not sure if you're guaranteeing all but the last blob is 100% full in this CL yet (haven't reveived deeply) but if not we may need to tweak this?

Reply:
Yeah the metrics definitely need improving. We could measure three things
* total size of all blobs together (currently)
* size of last blob (the leftover)
* number of blobs in a tx

Reply:
I changed the existing metric to record the size of the last left-over blob. The number of blobs can already be accessed via the metrics tracking the number of frames, and the total channel size is also already captured by a metric. So I consider the metrics issue resolved for now.